packages/shared_preferences/shared_preferences_android/android/src/test/java/io/flutter/plugins/sharedpreferences/SharedPreferencesTest.java

```diff
@@ -127,40 +153,40 @@ public void setBool() {
     final String key = "NewToFlutter";
     final boolean value = false;
     plugin.setBool(key, value);
-    Map<String, Object> flutterData = plugin.getAllWithPrefix("");
+    Map<String, Object> flutterData = plugin.getAllWithPrefix("", null);
     assertEquals(flutterData.get(key), value);
   }
 
   @Test
   public void clearWithPrefix() {
```

Comment:
Should the plugin call `clearWithPrefix` also take a list/set of allowed values?